docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  app:
+    build: .
+    image: gitlab-time-track
+    ports:
+      - 80:80
+    volumes:
+      - .:/srv/app
+    environment:
+      DB_HOST: mysql
+      DB_NAME: laravel_docker
+      DB_USER: app
+      DB_PASSWORD: password
+      SYMFONY_ENV: prod
```

Comment:
I still need to provide environment variables for GITLAB_SERVER_URL, GITLAB_TOKEN and SECRET_KEY. I'll do it later after we figure out stuff about the failing app server.